terra-env/apps.tf

```diff
@@ -74,3 +74,23 @@ module "workspace_manager" {
     google-beta.target = google-beta.target
   }
 }
+
+module "crl_janitor" {
+  source = "github.com/broadinstitute/terraform-ap-modules.git//crl-janitor?ref=crl-janitor-0.1.0"
```

Comment:
From what I see there is no module named crl-janitor in the terraform ap-deployments repo and no release with tag crl-janitor-0.1.0 is this referencing the correct repo/branch. I do see a module crl-test in terraform-ap-deployments. Not sure if this is the module you want though.